snapcraft.yaml

```diff
@@ -63,20 +70,29 @@ parts:
 
     third-party-modules:
       # Build PHP
-      - source: http://us1.php.net/get/php-7.0.2.tar.gz/from/this/mirror
+      - source: http://us1.php.net/get/php-7.0.5.tar.gz/from/this/mirror
         source-type: tar
         configflags:
+          - --enable-fpm
+          - --disable-pear
+          - --disable-phar
+          - --disable-phpdbg
```

Comment:
Enabling PHP-FPM, disabling unneeded things.
People won't be installing libs via PEAR or launch PHAR executables
We use xdebug and not phpdbg